src/atmos_param/socrates/interface/read_control.F90

```diff
@@ -9,8 +9,7 @@ MODULE read_control_mod
 
 ! Subroutine to set input algorithmic options for the core radiation code
 !------------------------------------------------------------------------------
-SUBROUTINE read_control(control, spectrum, do_cloud_simple)
-
+SUBROUTINE read_control(control, spectrum, do_clouds)
```

Comment:
I introduced this change both for clarity and to avoid duplicate code. With the way the two separate cloud schemes interact with the socrates modules, the namelist options `do_cloud_simple` and `do_cloud_spookie` would be passed through a number of the files. Where there is commonality between the two models as far as socrates is concerned (i.e. if any cloud scheme is wanted, do this), I renamed the existing variable where applicable, preserving the current structure in files such as `read_control.F90` without having to duplicate various statements. Does this make sense?